packages/server/test/unit/browsers/browsers_spec.js

```diff
@@ -5,7 +5,7 @@ const utils = require(`${root}../lib/browsers/utils`)
 const snapshot = require('snap-shot-it')
 
 const normalizeBrowsers = (message) => {
-  return message.replace(/(found are: ).*/, '$1chrome, firefox, electron')
+  return message.replace(/(found on your system are:)(?:\n- .*)*/, '$1\n- chrome\n- firefox\n- electron')
```

Comment:
I suspect the string generated changed format at some point, and because most developers have the same set of browsers installed, no one noticed with function wasn't working anymore.

I have chromium, but not chrome, installed on my machine, which meant the snapshot wasn't correct locally. This regex update fixes it.